src/screens/tenants/Tenants.tsx

```diff
@@ -41,6 +44,8 @@ export default class Tenants extends BaseScreen<Props, State> {
     await super.componentDidMount();
     const tenants = await this.centralServerProvider.getTenants();
     this.setState({ tenants });
+    const scanQRCode = Utils.getParamFromNavigation(this.props.route, 'openQRCode', this.state.createQrCodeTenantVisible);
+    this.setState({createQrCodeTenantVisible: scanQRCode})
```

Comment:
Please try to keep the coding style consistent with the style already used ({ value }) across your changes.